onnx/gen_proto.py

```diff
@@ -41,6 +41,14 @@ def process_ifs(lines, onnx_ml):
             elif (2 == in_if and not onnx_ml):
                 yield line
 
+
+PACKAGE_NAME_REGEX = re.compile(r'\{PACKAGE_NAME\}')
+
+def process_package_name(lines, package_name):
```

Comment:
Cool, this still happens in setup.py, right? We also need to call this in cmake.